custom_joint_state_publisher:
  joint_names: ["joint1", "joint2", "joint3", "joint4"]
  joint_positions: [-1, -1, -1, -1]